src/main/java/lotto/domain/Lotto.java

```diff
@@ -1,46 +1,34 @@
 package lotto.domain;
 
+import lotto.constant.ErrorMessage;
+
 import java.util.*;
-import java.util.stream.IntStream;
-import java.util.stream.Stream;
 
 public class Lotto {
-    private static final int LOWER_BOUND = 1;
-    private static final int UPPER_BOUND = 45;
-    private final Integer[] lotto;
-
-    public Lotto(Integer[] numbers) {
-        this.validateNumbers(numbers);
+    public static final int LOWER_BOUND = 1;
+    public static final int UPPER_BOUND = 45;
+    private static final int LOTTO_NUMBER_SIZE = 6;
+    private final List<Integer> lotto;
 
+    public Lotto(List<Integer> numbers) {
+        validateNumbers(numbers);
         this.lotto = numbers;
     }
 
-    public Lotto() {
-        this(generateLottoNumbers());
-    }
-
-    public Integer[] getLotto() {
+    public List<Integer> getLotto() {
         return lotto;
     }
 
-    private void validateNumbers(Integer[] numbers) {
-        if (numbers.length != 6) {
-            throw new IllegalArgumentException("로또 번호의 길이는 여섯 자리입니다.");
+    public static void validateNumbers(List<Integer> numbers) {
+        if (numbers.size() != LOTTO_NUMBER_SIZE) {
+            throw new IllegalArgumentException(ErrorMessage.LOTTO_NUMBER_LENGTH_ERROR);
         }
 
-        boolean isInRange = Arrays.stream(numbers).allMatch(number -> number >= LOWER_BOUND && number <= UPPER_BOUND);
+        boolean isInRange = numbers.stream().allMatch(number -> number >= LOWER_BOUND && number <= UPPER_BOUND);
```

Comment:
```suggestion
        boolean existInvalidNumber = numbers.stream().anyMatch(number -> number < LOWER_BOUND || number > UPPER_BOUND);
      
```

이런식으로 생각해보면 anyMatch도 사용할 수 있답니다 ㅎㅎ